python/setup.py

```diff
@@ -142,6 +148,8 @@ def cpp_flag(compiler):
     """Check and return the -std=c++11 compiler flag."""
     if has_flag(compiler, '-std=c++11'):
         return '-std=c++11'
+    elif sys.platform == 'win32':
```

Comment:
If MSVC has '-std=c++11' flag then control logic would never enter into this branch.

Reply:
Yep, good catch. I already have it fixed, just haven't pushed it yet.